### ISSUE TITLE
Will you  consider to publish this library to Maven or JitPack?

### ISSUE THREAD

**Issue by NeoLSN** [Doist/DateTimePicker#1]
This library is a very useful and awesome library.
Will you consider to publish this?

**Comment by goncalossilva** [comment]
Absolutely. We developed this internally and never really promoted it, so we just assumed no one else was using it. I'll add this to my to-do list 👍

**Comment by NeoLSN** [comment]
Wonderful news.
Hope it comes soon.

**Comment by mudar** [comment]
Hi @goncalossilva,
My comment is a bit late, but a quick solution would be to use jitpack:
https://jitpack.io/#Doist/DateTimePicker/e46a189880

I'm not that familiar with it, but I think using github releases makes it even more easier.

and thanks for the work you've done here :-)

**Comment by goncalossilva** [comment]
@mudar Thanks! That should work right out of the box.

We know this is late, but we still plan to publish this on bintray soon (it's on our roadmap for March, but things change).

**Comment by NeoLSN** [comment]
Is there any new progress?
